server/crawler/crawler.go

```diff
@@ -395,7 +407,7 @@ func extractKurtosisPackageContent(
 
 	nowAsUTC := getTimeProtobufInUTC()
 
-	kurtosisYamlFilePath := fmt.Sprintf("%s%s", packageRepositoryMetadata.RootPath, packageRepositoryMetadata.KurtosisYamlFileName)
+	kurtosisYamlFilePath := fmt.Sprintf("%s/%s", packageRepositoryMetadata.RootPath, packageRepositoryMetadata.KurtosisYamlFileName)
```

Comment:
@leoporoli This is surprising. i wonder how this is not causing bugs. What was the result before and after this change? It's possible we need to make a chagne in the frontend as well to handle this change?